entity-framework/core/providers/provider-log.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Log of provider-impacting changes - EF Core
+author: ajcvickers
+ms.author: avickers
+ms.date: 08/08/2018
+ms.assetid: 7CEF496E-A5B0-4F5F-B68E-529609B23EF9
+ms.technology: entity-framework-core
+uid: core/providers/provider-log
+---
+
+# Provider-impacting changes
+
+This page contains links to pull requests made on the EF Core repo that may require authors of other database providers to react. The intention is to provide a starting point for authors of existing third-party database providers when updating their provider to a new version.
+
```

Comment:
Maybe mention the Compliance tests somewhere (I know they are pre 2.2, but)

Reply:
I think this should go into the "writing a provider" piece, after which I mad add another link from here.